live/en/getting-here.html

```diff
@@ -203,14 +203,14 @@ <h3 id="car">By car</h3>
       <div class="col-md-6 text-justify">
         <h3>Public transport tickets</h3>
         <p>
-        You can buy a ticket in yellow vending machines for tickets accepting euro coins, or in newsstands. There is a wide variety of tickets, you can buy a 90-cent ticket that is valid for 30 minutes or there is 24 to 72 hour tickets. You can change buses and trams freely with a valid ticket. You need to mark the ticket in a machine once you enter a bus or tram. Once the ticket is marked, it is valid for 30 minutes or whatever time you've purchased. You can change to another bus or tram (but do NOT mark the ticket again, only do it during the first ride). Students with ISIC cards can buy discounted tickets, more information on <a href="http://www.dpb.sk/en/passengers/ticket/" title="Bratislava Public Transport website" target="_blank">Bratislava Public Transport website</a>.
+        You can buy tickets from the yellow vending machines or kiosks. They accept Euro coins. Alternatively you can also purchase tickets in newsstands. There are a wide variety of tickets, you can buy a 90-cent ticket that is valid for only 30 minutes or there are 24 to 72 hour tickets. You can change buses and trams freely with any valid ticket. You need to stamp, (have the ticket validated)  in the machine once you enter the bus or tram. Once the ticket is stamped, it is valid for 30 minutes or whatever time frame you've purchased. You can change to another bus or tram (but do NOT mark the ticket again, only do it the first time you get on). Students with ISIC cards can buy discounted tickets, more information on <a href="http://www.dpb.sk/en/passengers/ticket/" title="Bratislava Public Transport website" target="_blank">Bratislava Public Transport website</a>.
```

Comment:
"or there are 24 to 72 hour tickets" -> "There are also 24-hour tickets and 72-hour tickets."